src/types/sponsors.ts

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+// import { File } from 'types/file.interface';
+// import { Location } from 'types/location.interface';
+// import { RoleInterface } from './role.interface';
+
+
+interface CustomFieldInput {
```

Comment:
Suggestion(not-blocking): I think in leads types we also have the same interface, we can separate this to custom fields types and export it so we don't repeat the same interface